vllm/config.py

```diff
@@ -2212,6 +2215,53 @@ class CompilationLevel:
     PIECEWISE = 3
 
 
+class InductorHashCache:
```

Comment:
tried to place this class into `vllm.compilation.backends` , but then needs to be lazily imported, and `pydantic` will complain.